### PR TITLE
Several docs error correction.

### PR DESCRIPTION
**Pull request by CrainCH**
- "server" instead of "servers"
- word "is" is missing
- duplicated "and"
- "Wizards" to lower case
- "IHelloWorldService" instead of "IHelloWorldFormService" (interface
IHelloWorldFormService does not exist)
- "HelloWorldService" instead of "HelloWorldFormService" (class
HelloWorldFormService does not exist)
- "[webapp] dev server.launch" instead of "server-dev.launch" (outdated)
- "[webapp] dev ui.launch" instead of "ui-html-dev.launch" (outdated)
- "TablePage" instead of "Page"
- "MyTablePage" instead of "MyPage"
- "MyTablePageData" instead of "MyPageData"
- rephrase sentence "are added that trigger..."
- "customize" instead of "customized"
- "column" instead of "columns"
- "Properties" instead of "Property" (like the text below)
- "functional" instead of "functioning"
- "step" instead of "steps"
- "Contacts" instead of "Contact"
- rephrase sentence "The Scout new entry..."
- "QuickAccess" instead of "Quick Access"
- "config.properties" instead of "config.property"
- "properties" instead of "property"
- rephrase sentence "Open address in your http://localhost:8082/
browser"
- word "the" is missing
- "PersonTablePage" instead of "PersonTable" (class PersonTable does not
exist)
- "which" instead of "on the"
- "Country" instead of "City"
- "PersonTablePage" instead of "PersonTable"
- rephrase sentence "should the look as the..."
- word "work" is missing
- rephrase sentence "We now look at how..."
- "Eclipse" instead of "Eclispe"
- "Contact Info" instead of "Details"
- "Notes" instead of "Comments"
- "as needed" instead of "as are needed"
- rephrase sentence "... can be removed also."
- "implementations" instead of "implementation"
- "statement" instead of "statment"
- "straightforward" instead of "straight forward"
- rephrase sentence "We can now use..."
- rephrase sentence "...that we can now..."
- "organization" instead of "organizaion"
- "files" instead of "file"

### CODE REVIEW

File: docs/adoc/SdkWizardPage.adoc [hunk 1]
```diff
@@ -34,7 +34,7 @@ The context can be derived from a package selected in the Package Explorer or fr
 
 Source Folder:: The source folder of the Maven client module used for the creation of the page. The default value is the `src/main/java` folder in the Maven client module.
 Package:: The Java package that will contain the page class. The Scout SDK will try to guess the package name from the current context and derive matching package names for the Maven shared module.
-Name:: The name of the page class. According to Scout conventions the class name ends with the suffix `Page`.
+Name:: The name of the page class. According to Scout conventions the class name ends with the suffix `TablePage`.
```
Comment (padyx) [inline docs/adoc/SdkWizardPage.adoc]: Partially correct: The suffix is "TablePage" (as suggested by the wizard) when the page is a subtype of `AbstractPageWithTable`. For `AbstractPageWithNodes` on the other hand, the suffix is "NodePage".
I'd suggest slightly rephrasing this as: "... ends with the suffic TablePage (for subclasses of AbstractPageWithTable) or NodePage (for AbstractPageWithNodes)"

**Review by padyx — COMMENTED** [review]
Thank you for your contribution to Scout!

This looks very good - I added one single comment about the naming convention that you should be able to see as a review comment. If you could fix this we can merge it afterwards.

**Comment by CrainCH** [comment]
Thank you Patrick. I fixed the issue.

**Comment by padyx** [comment]
I have merged your pull request and will rebuild the website as soon as I have the time.
After that, your changes will be visible on the website.

Thank you again for your contribution to Scout!

**Pull request merged**